Storage/Service/AbstractUrlStorage.php

```diff
@@ -2,193 +2,89 @@
 
 namespace EMS\CommonBundle\Storage\Service;
 
+use GuzzleHttp\Psr7\Stream;
 use Psr\Http\Message\StreamInterface;
-use Symfony\Component\Filesystem\Filesystem;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 abstract class AbstractUrlStorage implements StorageInterface
 {
 
-    /**
-     * Returns the base url of the storage service
-     * @return string
-     */
-    abstract protected function getBaseUrl(): string;
+   abstract protected function getBaseUrl(): string;
 
-    /**
-     * returns the a file path or a resource url that can be handled by file function such as fopen
-     */
-    protected function getPath(string $hash, ?string $cacheContext = null, bool $confirmed = true, string $ds = '/'): string
+    protected function getPath(string $hash, bool $confirmed = true, string $ds = '/'): string
     {
-        $folderName = $this->getBaseUrl();
+        $folderName = $this->getBaseUrl() . $ds . $confirmed ? substr($hash, 0, 3) : 'uploads';
 
-        if (!$confirmed) {
-            $folderName .= $ds . 'uploads';
-        }
-
-        //isolate cached files
```

Comment:
Why aren't we isolating the cache files anymore? (I like the simplification, but what is the impact?)

Reply:
Oh I just read your opening ticket ;) 
So php bin/console c:cl will erase the cached files?

Reply:
Nope, but it's up to you to share that folder in an empty dir or a share NFS, or whatever